### PR TITLE
Add test 'Filter shopping carts'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Filter shopping carts'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | TEST_PATH="functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js" URL_FO=yourShopURL npm run specific-test

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/21225)

### CODE REVIEW

File: tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+const day = (`0${today.getDate()}`).slice(-2);
+
+// Current month
+const month = (`0${today.getMonth() + 1}`).slice(-2);
+
+// Current year
+const year = today.getFullYear();
+
+// Date today format (yyy/mm/dd)
+const todayDate = `${day}/${month}/${year}`;
```
Comment (SimonGrn) [inline tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js]: This is not coherent. You say in the comment that you want a date with the format `yyyy/mm/dd` but create a date with the format `dd/mm/yyyy` ?

**Review by SimonGrn — COMMENTED** [review]
A small comment !

File: tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js (hunk 1 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/02_orders/05_shoppingCarts/01_filterShoppingCarts.js]: Thanks done :+1:

**Review by NeOMakinG — APPROVED** [review]
lgtm

**Review by SimonGrn — APPROVED** [review]

**Pull request merged**